themes/custom/az_barrio/css/style.css

```diff
@@ -212,6 +212,29 @@ span.field-label {
   font-weight: bold;
 }
 
+/* On mobile, limit pager number items and hide previous/next buttons. */
+@media screen and (max-width: 576px) {
+  .hide-pager-item-mobile {
```

Comment:
I think we can use these display utilities for this https://digital.arizona.edu/arizona-bootstrap/docs/2.0/utilities/display/#notation

Reply:
Thanks for the suggestion @trackleft! I believe we still need the `hide-pager-item-mobile` class since Bootstrap doesn't have a class for setting the display to `list-item` and we would need that for the xs breakpoint. However, I was able to replace the `hide-pager-ellipsis-desktop` class with `d-sm-none`.